nim-vars.el

```diff
@@ -415,8 +402,9 @@ Magic functions.")
   "Builtin functions copied from system.nim.
 But all those functions can not use in NimScript.")
 
-(defconst nim-operators
-  '( "`" "{." ".}" "[" "]" "{" "}" "(" ")" )
+(defconst nim-buildin-operators
```

Comment:
maybe you can delete this if it's not using. I'm not sure this is typo of builtin instead of buildin.

Reply:
it's a typo, and yes I can delete it.